src/main/kotlin/no/nav/familie/ef/mottak/api/ForvaltningController.kt

```diff
@@ -12,15 +18,36 @@ import java.util.UUID
 @RestController
 @RequestMapping(path = ["/api/forvaltning"])
 @ProtectedWithClaims(issuer = "azuread")
-class ForvaltningController(private val ettersendingService: EttersendingService) {
+class ForvaltningController(private val ettersendingService: EttersendingService, private val taskService: TaskService,) {
 
     @PostMapping("/ettersending/splitt")
     fun trekkUtVedleggFraEttersending(@RequestBody ettersendingVedleggId: EttersendingVedleggId): ResponseEntity<String> {
         val nyEttersendingId = ettersendingService.trekkUtEttersendingTilEgenTaskForVedlegg(ettersendingVedleggId.id)
         return ResponseEntity.ok("Opprettet ny ettersending med id: $nyEttersendingId")
     }
+
+    @PostMapping("/ettersending/nycallid")
+    fun nyCallIdPåEttersendingTask(@RequestBody taskId: TaskId): ResponseEntity<String> {
```

Comment:
```suggestion
    fun settNyCallIdPåTaskForEttersending(@RequestBody taskId: TaskId): ResponseEntity<String> {
```

Navnet nå får funksjonen nesten til å se ut som en Task, så synes dette navnet ble greiere :p